Cue/app/tabs/library/CardListView.js

```diff
@@ -13,6 +13,7 @@ import ListViewHairlineSeparator from '../../common/ListViewHairlineSeparator'
 type Props = {
   cards: Array<Card>,
   isFiltering: boolean,
+  flagCard?: (cardUuid: string, flag: boolean) => any,
```

Comment:
Like with `CardListViewRow`, since this is a callback, I would call this `onFlagCard` instead.